core/src/mledb/mledb-player/mledb-player.service.ts

```diff
@@ -77,4 +79,95 @@ export class MledbPlayerService {
         });
         return ttc.length > 0;
     }
+
+    async getSprocketUserByPlatformInformation(platform: MLE_Platform, platformId: string): Promise<User> {
+        const playerAccount = await this.playerAccountRepository.findOneOrFail({where: {platform, platformId}, relations: {player: true} });
+        const {discordId} = playerAccount.player;
+
+        if (!discordId) {
+            throw new Error(`No discord account found for player ${playerAccount.player.name}`);
+        }
+
+        const user = await this.userService.getUser({
+            where: {
+                user: {
+                    authenticationAccounts: {
+                        accountId: discordId,
+                        accountType: UserAuthenticationAccountType.DISCORD,
+                    },
+                },
+            },
+            relations: {
+                user: {
+                    authenticationAccounts: true,
+                    members: {
+                        players: {
+                            skillGroup: {
+                                game: true,
+                            },
+                        },
+                    },
+                },
+            },
+        });
+        if (!user) {
+            throw new Error(`No sprocket user found (${platform} | ${platformId})`);
+        }
+
+        return user;
+
+    }
+
+    async getSprocketPlayerByPlatformInformation(platform: MLE_Platform, platformId: string): Promise<Player> {
+        const playerAccount = await this.playerAccountRepository.findOneOrFail({where: {platform, platformId}, relations: {player: true} });
+        const {discordId} = playerAccount.player;
+
+        if (!discordId) {
+            throw new Error(`No discord account found for player ${playerAccount.player.name}`);
+        }
+
+        const user = await this.userService.getUser({
+            where: {
+                user: {
+                    authenticationAccounts: {
+                        accountId: discordId,
+                        accountType: UserAuthenticationAccountType.DISCORD,
+                    },
+                },
+            },
+            relations: {
+                user: {
+                    authenticationAccounts: true,
+                    members: {
+                        players: {
+                            skillGroup: {
+                                game: true,
+                            },
+                        },
+                    },
+                },
+            },
+        });
+
+        if (!user) {
+            throw new Error(`No sprocket user found (${platform} | ${platformId})`);
+        }
+
+        const member = user.members.find(m => m.organizationId === 1);
```

Comment:
orgId 1 is Sprocket, not MLE. Make 2 or config option like for previous comment?